python/ray/air/config.py

```diff
@@ -650,7 +649,9 @@ class RunConfig:
     verbose: Optional[Union[int, "AirVerbosity", "Verbosity"]] = None
     stop: Optional[Union[Mapping, "Stopper", Callable[[str, Mapping], bool]]] = None
     callbacks: Optional[List["Callback"]] = None
-    progress_reporter: Optional["ProgressReporter"] = None
+    progress_reporter: Optional[
+        "ray.tune.progress_reporter.ProgressReporter"  # noqa: F821
```

Comment:
need to use full path now since the name `ProgressReporter` is ambiguous to sphinx, with the addition of `ray.tune.experimental.output.ProgressReporter`

Reply:
@can-anyscale @justinvyu is this just a hack to get a correct link in the docs? Could this be done in a different way that doesn't break the typing?

For reference see https://github.com/omni-us/jsonargparse/issues/609#issuecomment-2445578084. Before I found this pull request I also asked the question here https://discuss.ray.io/t/why-the-invalid-type-for-progress-reporter-of-ray-train-runconfig/20178

Reply:
Maybe this #48439?